### ISSUE TITLE
CI tests for starknet-forge template

### ISSUE THREAD

**Issue by piotmag769** [foundry-rs/starknet-foundry#149]
It should include running:
- `forge test` (using forge github action)
- `scarb fmt --check`

**Comment by piotmag769** [comment]
Blocked by https://github.com/foundry-rs/starknet-foundry/issues/396

**Comment by piotmag769** [comment]
No longer relevant since we move the template to our repo in #515
